src/Congo.RazorPages/Pages/Cart.cshtml.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Threading.Tasks;
+using Congo.Contracts.Responses.Cart;
+using Congo.RazorPages.Services;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace Congo.RazorPages.Pages
+{
+    public class CartModel : PageModel
+    {
+        public CartResponse Cart;
+        private readonly ICartService _cartService;
+
+        public CartModel(ICartService cartService)
+        {
+            _cartService = cartService;
+        }
+        public async Task<IActionResult> OnGetAsync()
+        {
+            string cartId;
+            if (Request.Cookies.ContainsKey("cartId"))
+                Cart = await _cartService.GetCart(Guid.Parse(Request.Cookies["cartId"]));
+            else
+            {
+                cartId = (await _cartService.GetCart()).ToString();
+                Response.Cookies.Append("cartId", cartId);
```

Comment:
This isn't right, `GetCart` returns a `CartResponse` - if a `cartId` doesn't exist in the cookie then you should show a message like 'No items' or something.

You're sorta on the right track, we want this sort of logic for add to cart, but not here.

Reply:
Need to catch error if we somehow have an invalid cart id